contracts/abstract/ReaperBaseStrategy.sol

```diff
@@ -117,7 +117,7 @@ abstract contract ReaperBaseStrategyv3 is
         totalFee = 450;
         callFee = 1000;
         treasuryFee = 9000;
-        strategistFee = 2500;
+        strategistFee = 0;
```

Comment:
this is unnecessary, init() function will never be called again.

Reply:
if `strategistFee` is not already set to 0 in the proxy, you can either do it in the `completeV2Upgrade()` function, or do it manually after the upgrade is complete.

Reply:
I don't remember this being a part of my changes. I will check what the fee is and add it to the upgrade function if needed.